go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/go-logr/logr v1.4.1
-	github.com/medik8s/common v1.13.0
+	github.com/medik8s/common v1.15.1
```

Comment:
NIT: There is already a newer version of common https://github.com/medik8s/common/tags that we can use